### PR TITLE
ingested emnlp/acl/naacl 2021 video links

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by davidstap** [comment]
Hi, I realised I missed some videos and I have added these now, so makes sense to re-run your video ingestion script :)

File: bin/ingest_archive_video.py [hunk 1]
```diff
@@ -120,19 +143,26 @@ def update_xml(data_dir, collection_id, extention, xml_tree):
...
     '--video_dir',
-    default='/Users/xinruyan/Dropbox/naacl-2013/',
     help='Directory contains all videos need to be ingested',
 )
-def main(video_dir):
-    collection_ids = get_collection_ids(video_dir=video_dir)
+@click.option(
+    '-s',
+    '--anth_id_style',
+    default='new',
```
Comment (mjpost) [inline bin/ingest_archive_video.py:151]: You don't have to pass this information; Instead, just get the Anthology ID from the file name, and call `anthology.utils.is_newstyle_id()` ([link](https://github.com/acl-org/acl-anthology/blob/5f0e68184f61d175607aa732c99da995c4f02427/bin/anthology/utils.py#L37)).

Ideally you should refactor the code to handle this—you can infer it and then avoid having to pass it all over.

**Review by mjpost — APPROVED** [review]
I spot-checked a lot of videos and they check out. Please see the note about the code; if you can fix that, then I'm fine with the merge.

**Comment by mjpost** [comment]
On second thought, these changes are large enough—and the change to the script trivial enough—that we should just merge now, to avoid potential complexities later. We can simplify the script later if there is need.

**Comment by mjpost** [comment]
On second thought, these changes are large enough—and the change to the script trivial enough—that we should just merge now, to avoid potential complexities later. We can simplify the script later if there is need.

**Pull request merged**